testeng/resources/cleanup-python-code-create-pr.sh

```diff
@@ -54,7 +54,11 @@ do_one_repo () {
   echo "Running script to create PR..."
   cd "$WORKSPACE/testeng-ci"
   pip install -r requirements/base.txt
-  local message="$(cat <<EOF
+  if [ -s "$repo_dir/.git/cleanup-python-code-message" ]
+  then
+     local message=$(cat "$repo_dir/.git/cleanup-python-code-message")
+  else
+     local message="$(cat <<EOF
```

Comment:
What does this line do exactly, and do we want one or the other, or potentially both?

Reply:
I think for now we want one or the other. There isn't really a use case for making this too customizable yet, and doing so would require non trivial changes elsewhere (to allow for more options on the jenkins job)

Reply:
I still don’t know what this does.